### DISCUSSION TITLE
Consolidated Option Documentation

### DISCUSSION THREAD

**Discussion by 0x0D15** [dakanji/RefindPlus#116] [General]
### Suggested Feature

Would you consider creating a single page that details a consolidated view of both rEFInd options and RefindPlus options with adjustments made to the rEFInd docs as needed? Essentially an improved version of [this](https://www.rodsbooks.com/refind/configfile.html) page. It will make it much easier when setting up config files as only a single reference is needed instead of multiple. Additionally it would allow for there to be improvements to the documentation's structure/organization; for example a MacOS specific section that details all settings that are only applicable to it.

### Background Information

N/A

**Comment by dakanji** [comment]
Thanks for the suggestion. Yes, this will looked at at some point. 

Note though that [config.conf-sample](https://github.com/dakanji/RefindPlus/blob/GOPFix/config.conf-sample) should more or less already provide a single source of configuration information and is clearly separated into new features, changed features and unchanged features.
